### ISSUE TITLE
Resource '$n' does not have attribute 'rendered' for variable '$n.rendered'

### ISSUE THREAD

**Issue by jwaldrip** [hashicorp/terraform#6544]
On destroy of an terraform config in atlas, I get the following error:

```

6 error(s) occurred:

* Resource 'template_file.bastion-write_files' does not have attribute 'rendered' for variable 'template_file.bastion-write_files.rendered'
* Resource 'template_file.k8s-worker-write_files' does not have attribute 'rendered' for variable 'template_file.k8s-worker-write_files.rendered'
* Resource 'template_file.bastion-units' does not have attribute 'rendered' for variable 'template_file.bastion-units.rendered'
* Resource 'template_file.etcd-write_files' does not have attribute 'rendered' for variable 'template_file.etcd-write_files.rendered'
* Resource 'template_file.k8s-worker-units' does not have attribute 'rendered' for variable 'template_file.k8s-worker-units.rendered'
* Resource 'template_file.etcd-units' does not have attribute 'rendered' for variable 'template_file.etcd-units.rendered'
```

Each template_file resource looks like this (pretty simple):

``` hcl
resource "template_file" "etcd-write_files" {
  template = "${file("config/write_files/etcd.yml")}"
}
```

The template file itself:

```
${file("config/write_files/shared/bash-functions.yml")}
${file("config/write_files/shared/coredump.yml")}
${file("config/write_files/shared/journald-settings.yml")}
${file("config/write_files/internal/ssh-config.yml")}
```

**Comment by chadgrant** [comment]
yep +1

**Comment by mitchellh** [comment]
This should be fixed in TF 0.8 when you run a destroy plan.
